exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -194,7 +194,8 @@ func (e *exporter) sendBatch(ctx context.Context, lines []string) error {
 	resp, err := e.client.Do(req)
 
 	if err != nil {
-		return err
+		e.settings.Logger.Sugar().Errorf("failed to send request: %v", err)
```

Comment:
```suggestion
e.settings.Logger.Sugar().Error("Failed to send request", zap.Error(err))
```

Ideally there should be a simple static string pattern that can be matched within logging tools.